.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,22 +1,48 @@
 ---
-name: Bug Report
-about: Reporting issues with the VA API Platform
-
+name: 🔴 Bug Report
+about: For reporting issues with the VA API Platform
 ---
 
-## APIs Affected
+Important note: As this is a public forum, please do not include your VA API OAuth Client ID, VA API OAuth Client Secret, VA API key, or any Personally Identifiable Information (PII) or Protected Health Information (PHI) in your report.
+
+### Troubleshooting note
+* For test apps, please make sure you are using your registered URL, not the URL provided in the example.
+
+
+## Email address
+
+<!-- Please include your email address. -->
+
+
+## APIs affected
+
+<!-- Please list affected APIs (if applicable). -->
 
-<!-- Please list affected APIs -->
 
-## Brief Summary 
+## What I expected to happen
 
-<!-- One or two sentence summary of the issue. --> 
+<!-- What did you expect to happen? --> 
 
-## Severity of the Issue
+
+## What actually happened
+
+<!-- Describe in detail what went wrong. Screenshots, gifs, and videos are encouraged. --> 
+
+
+## Severity of the issue
 
 <!-- Please indicate how severe this issue is for your use case. -->
-<!-- Does it prevent you from using the API? Were you able to workaround? -->
 
-## Steps to Reproduce
+[ ] I am unable to use the API
+[ ] I was able to use the API, but am not getting the results I expected
+[ ] I was able to workaround the issue
```

Comment:
Nice! Dig it.

Reply:
👍

Reply:
Thanks for adding these @juliaelman.  I think these definitions of severity make perfect sense for developers who are still in active development (.i.e. pre-launch of their down product) as they are very "development" focused.  Once they do launch, does capturing the magnitude of the impact and the number of end users impacted matter more as there own development is not really impacted in this case?